### DISCUSSION TITLE
Ungodly frustration with GitHub mobile

### DISCUSSION THREAD

**Discussion by mkultracreative** [community/community#72109] [Mobile]
Please explain why I would ever use the mobile app. A logical point to its benefit over, any other option, at all. 

I'm flabbergasted by simple reductions to its functionality such as managing repos. Deleting. Cloning. Anything that isn't basic markdown or forced attempts at some awkward developer style social media ploy. The app only ever causes more confusion as to what its purpose is the more you attempt to make sense of it.

**Comment by elisealix22** [comment]
Hi @pixelpandacreative , thanks for your feedback. Let us know if there are any features or improvements that would make the app more useful for you. We're always open to suggestions.
